packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -89,13 +89,19 @@ const messages = {
             LOCATION_UNDELIVERABLE: {
                 title: 'The restaurant doesn\'t deliver to this address',
                 message: 'Search for a different restaurant or change to a collection order.',
-                buttonText: 'OK'
+                buttonText: 'Ok'
```

Comment:
Did you mean to change the casing here?

Reply:
Yeah, since I wanted to match what the iOS app shows, but thanks for checking 👍